doc/source/train/user_guide.rst

```diff
@@ -652,28 +716,127 @@ Checkpoints can be loaded into the training function in 2 steps:
 2. The checkpoint to start training with can be bootstrapped by passing in a
    ``checkpoint`` to ``trainer.run()``.
 
-.. code-block:: python
+.. tabs::
 
-    from ray import train
-    from ray.train import Trainer
+  .. group-tab:: PyTorch
```

Comment:
Suggestion: add `:emphasize-lines: <line numbers>` here to highlight the lines added.